### DISCUSSION TITLE
Reading WebRTC/Live stream from python

### DISCUSSION THREAD

**Discussion by mariia-artamonova** [ant-media/Ant-Media-Server#5953] [FAQs] [answered]
We have a use case to read the WebRTC stream from python, is there any specific libraries?. Our objective is to read live stream from python which is required for some AI modules

**Comment by ZohaibDurrani** [comment, marked as answer]
> We have a use case to read the WebRTC stream from python, is there any specific libraries?. Our objective is to read live stream from python which is required for some AI modules   

Use `python-socket` I have done this very similar and deployed micro controllers on the sites. Where we are receiving the live stream and sending to our server and then server processes the stream and send the predictions on the dashboard.
